src/Components/Category/Category.js

```diff
@@ -5,6 +5,11 @@ import { CategoryCard } from "./CategoryCard";
 
 export const Category = () => {
   const [category, setCategory] = useState([]);
+
+  useEffect(() => {
+    document.title = "Home | Echo Store";
+  }, []);
```

Comment:
you can make custom hook for changing document title

Reply:
done, Thanks